tsconfig.json

```diff
@@ -14,7 +18,7 @@
 		"outDir": "./dist",
 		"strict": true,
 		"strictNullChecks": true,
-		"target": "es2017",
+		"target": "es5",
```

Comment:
I would suggest updating all to es2017. This is supported in Node versions greater than 8.10.x Same  for other modules.

Reply:
Happy to update Orizuru to es2017. Makes sense as the node LTS is now 10.15.1.